app/services/exports/lettings_log_export_service.rb

```diff
@@ -1,69 +1,27 @@
 module Exports
-  class LettingsLogExportService
+  class LettingsLogExportService < Exports::XmlExportService
     include Exports::LettingsLogExportConstants
     include CollectionTimeHelper
 
-    def initialize(storage_service, logger = Rails.logger)
-      @storage_service = storage_service
-      @logger = logger
-    end
-
     def export_xml_lettings_logs(full_update: false, collection_year: nil)
-      start_time = Time.zone.now
-      daily_run_number = get_daily_run_number
       archives_for_manifest = {}
-      recent_export = LogsExport.order("started_at").last
+      recent_export = Export.order("started_at").last
```

Comment:
This might also need to depend on collection now?

Reply:
Yes, I was a bit reluctant to add it into the loop, cause I was worried it would always grab the newly created one, but if we're filtering by collection that should be fine